src/makeCalculator.js

```diff
@@ -4,7 +4,31 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {};
+
+  calculator.result = 0;
+
+  calculator.operate = (callback, number) => {
+    calculator.result = callback(number);
+
+    return calculator;
+  };
+
+  calculator.add = (a) => calculator.result + a;
+
+  calculator.subtract = (a) => calculator.result - a;
+
+  calculator.multiply = (a) => calculator.result * a;
+
+  calculator.divide = (a) => calculator.result / a;
+
+  calculator.reset = () => {
+    calculator.result = 0;
+
+    return calculator;
+  };
```

Comment:
Same as above, this is a code knowledge issue. When creating a method in an object, you don't need to use the function keyword, use the shortcut instead. So, you should replace the arrow function with a method definition like this: `reset() {...}`.